### PR TITLE
Update TinkerForge Readme.md

### PR DESCRIPTION
**Pull request by sihui62**
Added explanation for using symbolic names if more than one device of the same type is used.
Related community forum thread:
https://community.openhab.org/t/how-to-use-multiple-tinkerforge-io-16-bricklet/36749

Signed-off-by: Siegfried Huismann <sihui@gmx.de>

### CODE REVIEW

**Comment by Confectrician** [comment]
Hello and thanks for your contribution! 👋 
Many parts of the documentation are [imported from other openHAB repositories](https://github.com/openhab/openhab-docs#automatically-generated-parts), e.g., the Binding- or add-on specific repository. This way we ensure that the articles are always in sync with the actual source code. This method has proven to be quite helpful to keep documentation up-to-date.
Long story short, you'll need to propose your change in the correct repository: 
https://github.com/openhab/openhab1-addons/blob/master/bundles/binding/org.openhab.binding.tinkerforge/README.md

Thanks again for your willingness to contribute! 🙇
Please close this PR after you've created the new one.

**Comment by sihui62** [comment]
Thx.
Moved to https://github.com/openhab/openhab1-addons/pull/5376
